.github/workflows/dockerhub.yml

```diff
@@ -14,6 +14,7 @@ on:
         description: "Optional: List of Additional Build Contexts"
         type: "string"
         required: false
+        default: .
```

Comment:
This is to avoid unintentional use of git context for docker builds (https://github.com/docker/build-push-action?tab=readme-ov-file#git-context)